plugins/woocommerce/changelog/2022-11-24-08-07-11-448089

```diff
@@ -0,0 +1,4 @@
+Significance: major
+Type: enhancement
+
+Fix deprecated error in PageController ini PHP v8.0
```

Comment:
Small typo ("ini") and, if I read the PR description correctly, this is actually something that happens [as of PHP 8.1](https://www.php.net/releases/8.1/en.php#deprecations_and_bc_breaks) rather than PHP 8.0. Taking those things into consideration, how about:

```suggestion
Avoid deprecation notices under PHP 8.1 when calling wp_parse_url().
```